hashicorp/dev-portal
# feat: user profile bookmarks stub

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-ksuser-profile-stub-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1199634971449915/1202841857338448) 🎟️

## 🗒️ What

This PR stubs out the main content of the user profile bookmarks page. 

It renames `/profile` 👉 `/profile/bookmarks`, since there will eventually be a proper base profile page. All bookmarks are fetched and rendered in a grid without pagination. 

## 🧪 Testing

- Pull down this branch, run things locally.
- Log in, open the user dropdown and navigate to the bookmarks page. 
- The grid of your bookmark data should render. 

<img width="1505" alt="Screen Shot 2022-08-22 at 10 23 00 AM" src="https://user-images.githubusercontent.com/36613477/185981833-48119330-b515-495d-aa45-8b362a86711a.png">


## 💭 Anything else?

- Pressing the bookmark button on the cards now should only trigger a console log. In a follow-up PR I will add the confirmation dialog that actually updates the data. 
- The sidebar layout still needs to be implemented. 
- The base empty state still needs to be implemented.
